platforms/software/build-init/src/integTest/groovy/org/gradle/buildinit/specs/internal/BuildInitSpecsInteractiveIntegrationTest.groovy

```diff
@@ -18,13 +18,14 @@ package org.gradle.buildinit.specs.internal
 
 import org.gradle.buildinit.plugins.AbstractInteractiveInitIntegrationSpec
 import org.gradle.buildinit.plugins.TestsBuildInitSpecsViaPlugin
+import org.gradle.buildinit.plugins.fixtures.WrapperTestFixture
 import org.gradle.integtests.fixtures.executer.GradleHandle
 import org.gradle.test.fixtures.ConcurrentTestUtil
 import org.gradle.test.fixtures.file.LeaksFileHandles
 import org.gradle.util.internal.TextUtil
 
+@LeaksFileHandles
 class BuildInitSpecsInteractiveIntegrationTest extends AbstractInteractiveInitIntegrationSpec implements TestsBuildInitSpecsViaPlugin {
-    @LeaksFileHandles
```

Comment:
💭 This might still be neededs on Windows CI.

Reply:
I put it on the class since there were two tests and I wasn't sure if it would be needed for both